### PR TITLE
Adding text to admin

### PR DESCRIPTION
**Pull request by lschatzkin**
Text updates to the admin areas that are currently built out. Added a css tooltip

To do:

- congress actions
- add logo
- mailing list
- status options

### CODE REVIEW

File: app/assets/stylesheets/admin/base.scss [hunk 1]
```diff
@@ -1,6 +1,42 @@
...
\ No newline at end of file
+}
+
+/* Tooltip container */
+.tooltip {
+  color: $gray;
+  border: 2px solid $gray;
+  border-radius: 1rem;
+  display: inline-block;
+  font-size: 18px;
```
Comment (wioux) [inline app/assets/stylesheets/admin/base.scss]: I've been sticking to em/rem units for font-size (16px = 1rem).

**Comment by wioux** [comment]
There's not a super clear method to the style organization now, but I'm trying to keep most of the general rules in admin.scss. Could you move the tooltip styles there? I've been putting variables and colors in mixins.scss, so that's where $eff-red/$gray should go.

Thanks for starting on this!

File: app/assets/stylesheets/admin/base.scss (hunk 1 above)
Comment (lschatzkin) [reply app/assets/stylesheets/admin/base.scss]: Oh, me too, thanks for catching that.

**Comment by lschatzkin** [comment]
Is there a way to share the variables.scss file in /application, or do we need to just recreate them? I'm thinking about the colors but it's not a big deal.

I keep usually keep global variables in variables.scss, I'm happy to reorganize the files if that works for you.

File: app/assets/stylesheets/admin/admin.scss [hunk 2]
```diff
@@ -47,3 +47,38 @@
 #epiceditor {
```
Comment (wioux) [inline app/assets/stylesheets/admin/admin.scss]: Aahh I didn't realize this file still exists, I plan to delete it. The admin styles are in `app/assets/stylesheets/admin.scss`.

Comment (lschatzkin) [reply app/assets/stylesheets/admin/admin.scss]: oops!

**Comment by wioux** [comment]
I'd like to keep the themes separate and the number of variables on the admin side pretty small. The light theme does well with a smaller color palette IMO.

**Comment by lschatzkin** [comment]
> I'd like to keep the themes separate and the number of variables on the admin side pretty small. The light theme does well with a smaller color palette IMO.

Okay, works for me.

**Pull request merged**